app/controllers/course/leaderboards_controller.rb

```diff
@@ -3,6 +3,7 @@ class Course::LeaderboardsController < Course::ComponentController
   before_action :check_component
   before_action :load_settings
   before_action :add_leaderboard_breadcrumb
+  helper Course::Achievement::ControllerHelper.name.sub(/Helper$/, '')
```

Comment:
Just to double check, are you asking if we should instead include a Common Module in the `leaderboards_controller`, and include all the necessary helpers in that Common Module? Or am I understanding this wrongly?